sage-ios/SAGE/SAGE/LoginController.swift

```diff
@@ -66,16 +71,21 @@ class LoginController: UIViewController {
                     (valid: Bool) -> Void in
                     dispatch_async(dispatch_get_main_queue(), { () -> Void in
                         if (valid) {
-                            if let verified = LoginOperations.getUser()?.verified {
-                                if verified {
+                            LoginOperations.getState({ (user, currentSemester, userSemester) -> Void in
+                                KeychainWrapper.setObject(user, forKey: KeychainConstants.kUser)
```

Comment:
you dont need to do any of this keychain stuff anymore since the getState function does this for you now